proto/spire/api/server/agent/v1/agent.proto

```diff
@@ -190,3 +197,13 @@ message AgentX509SVIDParams {
     // ignored. The agent X509-SVID attributes are determined by the server.
     bytes csr = 1;
 }
+
+message PushStatusRequest {
+    // Required. Serial number of the bundle used to sign Agent identity.
+    string authority_serial = 1;
+}
+
+message PushStatusResponse {
+    // Keys that are no longer secure and must be rotated.
+    repeated bytes taintedKeys = 1;
```

Comment:
The intention here is to return the list of tainted keys (JWT or X.509) so agent can react renewing all affected SVIDs (agent SVID, cached workload X.509 SVIDs, cached JWT SVIDs)

Reply:
for now it is a simple list, and we verify if that key was used for jwt or x509, and taint any of them, 
Maybe I overthink this, and we can just create 2 lists and didn't verify if the key was used by only one of our bundles